java/ql/src/experimental/Security/CWE/CWE-094/GroovySandbox.java

```diff
@@ -0,0 +1,91 @@
+public class GroovySandbox {
+    public static final String WHITELIST_PATTERNS = "sandboxing.whitelist.patterns";
+    public static final String VAR_TYPES = "sandboxing.variable.types";
+
+    public static final ThreadLocal<Map<String, Object>> COMPILE_OPTIONS = new ThreadLocal<Map<String, Object>>();
+
+    
+    public static void main(String[] args) {
+        CompilerConfiguration conf = new CompilerConfiguration();
+        ImportCustomizer customizer = new ImportCustomizer();
+        customizer.addStaticStars("java.lang.Math");
+        ASTTransformationCustomizer astcz = new ASTTransformationCustomizer(
+                singletonMap("extensions", 
+                singletonList("SandboxingTypeCheckingExtension.groovy")),
+                CompileStatic.class);
+        conf.addCompilationCustomizers(astcz);
```

Comment:
This example is really quite long. Can you abbreviate it with `// ...` comments to keep just the essential parts for illustrating your point?